### ISSUE TITLE
Unit tests and continuous integration

### ISSUE THREAD

**Issue by femtotrader** [mhallsmoore/qsforex#19]
Hello,

it will be a good idea to add some unit tests and continuous integration (Travis might help)

Kind regards

**Comment by mhallsmoore** [comment]
Indeed, right now there are only tests for portfolio.py and position.py. They are not in a CI framework either. Travis is a good idea. I'll look into it!
